extensions/amp-story/0.1/progress-bar.js

```diff
@@ -52,13 +53,16 @@ export class ProgressBar {
     this.root_ = null;
 
     /** @private {number} */
-    this.pageCount_ = 0;
+    this.segmentCount_ = 0;
```

Comment:
was your intention to rename all `page` to `segment` ?

I saw now it's a mix in this file.

Reply:
+1; I like the naming "segment" better than "page" here, but we should make sure it's consistent.

Reply:
Yes. I cleaned this up. I drew the new line at the `ProgressBar` class, so everything in that file is now a `segment` while `system-layer` and `amp-story` are still using `pages`